mathjax3-ts/output/chtml/Wrappers/math.ts

```diff
@@ -69,9 +75,20 @@ export class CHTMLmath<N, T, D> extends CHTMLWrapper<N, T, D> {
      */
     public toCHTML(parent: N) {
         super.toCHTML(parent);
-        if (this.node.attributes.get('display') === 'block') {
-            this.adaptor.setAttribute(this.chtml, 'display', 'true');
-            this.adaptor.addClass(parent, 'MJX-DISPLAY');
+        const chtml = this.chtml;
+        const adaptor = this.adaptor;
+        const attributes = this.node.attributes;
+        const display = (attributes.get('display') === 'block');
```

Comment:
Why parentheses?

Reply:
I've always preferred them when assigning a variable the result of a boolean operation.  It makes the order of operations explicit.  I can remove if you think it is important, but we discussed this once before, and you indicated that it was OK and that we could make it part of our style definition if I wanted.